test/ctr.bats

```diff
@@ -1284,7 +1284,6 @@ function wait_until_exit() {
 	[ "$status" -eq 0 ]
 	[[ "$output" =~ "209715200" ]]
 
-
```

Comment:
The patch content do not align with a description. Not sure why have this at all

Reply:
fixed!